### PR TITLE
Remove Max Limits per Container and Raise Default CPU and Memory Limits

### PR DESCRIPTION
**Pull request by tobru**
## Checklist

- [X] PR contains a single logical change (to build a better changelog).
- [X] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

**Review by simu — APPROVED** [review]

**Review by bastjan — COMMENTED** [review]
Do we really want default limits this high? A user setting no limits can have a maximum of 8 pods with those limits.

**Comment by tobru** [comment]
We can lower them a bit if you think so. The idea behind higher limits is that this guarantees that 80% of the user workload just works, even if the user doesn't tweak them. But I'd be OK to go down to 0.75 CPU and 768Mi...

File: class/defaults.yml [hunk 1]
```diff
@@ -117,16 +117,13 @@ parameters:
...
-            cpu: "1"
-            memory: "4Gi"
           min:
             cpu: "10m"
             memory: "4Mi"
             ephemeral-storage: "100Ki"
           default:
-            cpu: "200m"
-            memory: "200Mi"
+            cpu: "1"
```
Comment (ccremer) [inline class/defaults.yml:125]: I've rarely seen apps or charts setting a limit to 0.5 or higher. A high-demanding workload usually has resources defined, so a really-high default limit is probably not needed
```suggestion
            cpu: "600m"
```

**Comment by tobru** [comment]
I've lowered the default limit as discussed.

**Review by bastjan — APPROVED** [review]

**Review by ccremer — APPROVED** [review]

**Pull request merged**